src/main/java/com/thoughtworks/gauge/screenshot/ICustomScreenshot.java

```diff
@@ -0,0 +1,4 @@
+package com.thoughtworks.gauge.screenshot;
+
+public interface ICustomScreenshot {
```

Comment:
This is not java syntax. The use of `I` before an interface is in C# and not Java. Please change this to CustomScreenshot